pyproject.toml

```diff
@@ -16,8 +16,10 @@ skip_glob = ["tools/*", "exampleCourse/*", "testCourse/*", "node_modules/*"]
 [tool.pyright]
 include = [
     "./elements/pl-hidden-hints",
-    "./elements/pl-order-blocks",
```

Comment:
IIRC we should either add `--skipunannotated` or remove this, but not both - is that correct?

Reply:
Not quite; the issue is that if we add ` --skipunannotated` (which we should to have typechecking for annotated functions in `prairielearn.py`), then order blocks won't have any typechecking done (since there aren't any annotations). So it's actually ok to leave it in the include, but pyright won't do anything.

Although this technically disables a dev tool for order blocks, those annotations need to be added back anyway, and the type inference that can be done without annotations is super limited.